src/views/SignUp/SignUp.js

```diff
@@ -0,0 +1,145 @@
+import React,{Component} from "react";
+import Layout from '../../components/Layout';
+
+export default class SignUp extends Component{
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      Lastname: '',
+      email: '',
+      username:'',
+      password:''
+    };
+
+    this.handleSubmitFields = this.handleSubmitFields.bind(this);
+    this.VerifyString = this.VerifyString.bind(this);
+    this.VerifyEmail = this.VerifyEmail.bind(this);
+    this.verifyPassword = this.verifyPassword.bind(this);
+    this.comparePassword = this.comparePassword.bind(this);
+  }
+
+  handleSubmitFields(event) {
+
+    if(this.state.includes(null)){
+      return;
+    }
+
+    var url = 'https://localhost:5001/api/workers/';
+
+    fetch(url, {
+      method: 'POST',
+      body: JSON.stringify(this.state),
+      headers:{
+        'Content-Type': 'application/json'
+      }
+    }).then(res => res.json())
+    .then(response => console.log('Success:', JSON.stringify(response)))
+    .catch(error => console.error('Error:', error));
+    event.preventDefault();
+  }
+
+  VerifyString({ target }) {
+    const {name} = target;
+    if(name.length<=0){
+      console.log('is empty')
```

Comment:
Remove all `console.log`, they should not be committed as their only purpose is for debugging.

Reply:
7ed41c2
1 changed file